src/services/resource.js

```diff
@@ -39,17 +42,34 @@ module.exports = class resourceHelper {
 			let filter = {
 				organization_id,
 			}
+			let uniqueInReviewResourcesIds = []
+			let reviewer_notes = {}
+			let in_progress_resource_ids = {}
+			let rejected_resource_ids = {}
+			let rejected_and_resported_resource_ids = {}
+			let showNotes = false
 			// fetch the details of resource and organization from resource creator mapping table by the user
 			const resource_creator_mapping_data = await resourceCreatorMappingQueries.findAll({ creator_id: user_id }, [
 				'resource_id',
 				'organization_id',
 			])
 
+			if (resource_creator_mapping_data.length <= 0) {
+				return responses.successResponse({
+					statusCode: httpStatusCode.ok,
+					message: 'RESOURCE_LISTED_SUCCESSFULLY',
+					result,
+				})
+			}
 			// get the unique resource ids from resource creator mapping table by the user
-			const uniqueResourceIds = [...new Set(resource_creator_mapping_data.map((item) => item.resource_id))]
+			const uniqueResourceIds = utils.getUniqueElements(
+				resource_creator_mapping_data.map((item) => item.resource_id)
+			)
 
 			// get the unique organization ids from resource creator mapping table by the user
-			const OrganizationIds = [...new Set(resource_creator_mapping_data.map((item) => item.organization_id))]
+			const OrganizationIds = utils.getUniqueElements(
+				resource_creator_mapping_data.map((item) => item.organization_id)
+			)
 
 			if (queryParams[common.TYPE]) {
```

Comment:
@adithyadinesh0412  why its is splitting the type, type can be single string right ?